src/main/java/org/beanpod/switchboard/entity/OutputChannelEntity.java

```diff
@@ -31,17 +30,14 @@
   @Column(name = "id")
   private Long id;
 
-  @OneToOne(
-      optional = false,
-      cascade = {CascadeType.MERGE})
+  @OneToOne(cascade = {CascadeType.MERGE})
   @JoinColumn(name = "channel_id")
   private ChannelEntity channel;
 
   @ManyToOne(
-      optional = false,
```

Comment:
Same comment here, why are we removing this?

Reply:
I agree with you, however if i have the optional tag, I would get an error like: not-null property references a null or transient value.
This is part of the workaround solution where the logic doesn't really meet the code.